docs/08-01-provisioning-kyma-environment.md

```diff
@@ -1,6 +1,6 @@
-# Provision Kyma Runtime using KEB
+# Provision SAP BTP, Kyma runtime using KEB
 
-This tutorial shows how to provision Kyma Runtime on Azure using Kyma Environment Broker.
+This tutorial shows how to provision SAP BTP, Kyma runtime on Azure using Kyma Environment Broker.
 
 ## Prerequisites
```

Comment:
1. The first link in the Prerequisites is broken, and the second one leads to the wrong page.
2. Please change all instances of Kyma Environment Broker to KEB (except the first one, of course)